iOS/airBnb/airBnb/ViewController/LocationSearch/LocationSearchViewController.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  LocationSearchViewController.swift
+//  airBnb
+//
+//  Created by HOONHA CHOI on 2021/05/19.
+//
+
+import UIKit
+
+class LocationSearchViewController: UITableViewController {
+    
+    private var cities = ["서울","광주","의정부시","수원시","대구","울산","부천시"]
```

Comment:
Dummy 인가요? 어떤 의도인지, 아니면 제거되어야 할 대상인지 자신 뿐만 아니라 코드를 읽는 사람을 위해서 주석을 활용하면 좋을거 같아요🙂

```suggestion
    // TODO: - 더미 데이터. 제거 대상
    private var cities = ["서울","광주","의정부시","수원시","대구","울산","부천시"]
```